assets/queries/cloudFormation/BucketPolicyEnabled/test/negative.yaml

```diff
@@ -0,0 +1,66 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: A sample template
+Resources:
+  mybucket:
+    Type: "AWS::S3::Bucket"
+    DeletionPolicy: Retain
+    Properties:
+      ReplicationConfiguration:
+        Role:
+          "Fn::GetAtt":
+            - WorkItemBucketBackupRole
+            - Arn
+        Rules:
+          - Destination:
+              Bucket:
+                "Fn::Join":
```

Comment:
Linter is complaining about this, Try removing the surrounding double quotes: `Fn::Join`